<ui version="4.0" >
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QComboBox" name="searchPlayerComboBox" >
         <property name="currentIndex" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer4" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>442</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>Depth Limit</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="depthLimitBox" >
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>100000</number>
         </property>
         <property name="singleStep" >
          <number>200</number>
         </property>
         <property name="value" >
          <number>1200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel2" >
         <property name="text" >
          <string>Initial Depth Limit</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="initialDepthLimitBox" >
         <property name="maximum" >
          <number>100000</number>
         </property>
         <property name="singleStep" >
          <number>200</number>
         </property>
         <property name="value" >
          <number>400</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel3" >
         <property name="text" >
          <string>Deepning Step</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="deepningStepBox" >
         <property name="maximum" >
          <number>10000</number>
         </property>
         <property name="value" >
          <number>200</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel4" >
         <property name="text" >
          <string>Node Limit</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nodeLimitBox" >
         <property name="maximum" >
          <number>999999999</number>
         </property>
         <property name="singleStep" >
          <number>10000</number>
         </property>
         <property name="value" >
          <number>1600000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel5" >
         <property name="text" >
          <string>Table Size Limit</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="tableSizeLimitBox" >
         <property name="maximum" >
          <number>999999999</number>
         </property>
         <property name="singleStep" >
          <number>10000</number>
         </property>
         <property name="value" >
          <number>5000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel6" >
         <property name="text" >
          <string>Table Record Limit</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="tableRecordLimitBox" >
         <property name="minimum" >
          <number>-10</number>
         </property>
         <property name="maximum" >
          <number>400</number>
         </property>
         <property name="value" >
          <number>200</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel7" >
         <property name="text" >
          <string>Search &amp;Time</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>searchTimeBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="searchTimeBox" >
         <property name="maximum" >
          <number>1200</number>
         </property>
         <property name="value" >
          <number>120</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>330</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>Multi PV Width</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>multiPVWidthBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="multiPVWidthBox" >
         <property name="maximum" >
          <number>3000</number>
         </property>
         <property name="singleStep" >
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer5" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="defaultButton" >
         <property name="text" >
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton" >
         <property name="text" >
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer1" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>237</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>&amp;OK</string>
         </property>
         <property name="shortcut" >
          <string>Alt+O</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>&amp;Cancel</string>
         </property>
         <property name="shortcut" >
          <string>Alt+C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
